multiqc/utils/software_versions.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python
 
-""" Super Special-Case MultiQC module to produce report section on software versions """
```

Comment:
Haha, I didn't notice that docstring 😅

Reply:
Confused me for a moment, I thought that the file had been duplicated as it also had the same filename 😅